packages/fxa-settings/src/pages/AccountRecoveryResetPassword/en.ftl

```diff
@@ -0,0 +1,8 @@
+password-link-expired = Reset password link expired
+password-link-damaged = Reset password link damaged
+create-new-password = Create new password
+receive-new-link = Receive new link
```

Comment:
```suggestion
# Link that user can click to receive a new reset password link
receive-new-link = Receive new link
```

Reply:
Comment added to provide more context for localizers, since this is an action rather than something like header text.